CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 The crates in this repository do not adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) at this time.
 
+## [4.0.2]
+
+### Changes
```

Comment:
:wrench: kind of a grey area as I think "fixed" are usually customer facing, but having the sentence worded as "fix" seems out of place
```suggestion
### Fixed
```

Reply:
I don't think we've used "Fixed" in the past. or at least the past few releases.

Reply:
Never mind, I didn't look back far enough.